templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -66,49 +41,14 @@ message](/help/quote-and-reply).
 
 {end_tabs}
 
-### Get a link to a specific topic
+## Get a link to a specific topic
```

Comment:
I think the previous structure was correct, as the instructions describe how to get a topic link to use anywhere/outside of Zulip. The best way to link to a topic within Zulip is by the method above on this page.

Reply:
I don't see it that way. "Link to Zulip from anywhere" is applicable for all views, whether that be a list of messages, a user profile, or even the help files; and instructs the user to copy the URL from the address bar.

"Get a link to a specific message/topic" are not applicable to all views, but only the topic/message views; and provide a menu-driven method to get an URL. They are not sub-categories of "link from anywhere": they are their own stand-alone categories of action.